shared/reducers/profile.js

```diff
@@ -75,6 +75,8 @@ export default function(state: Types.State = initialState, action: ProfileGen.Ac
       })
     case ProfileGen.updatePgpPublicKey:
       return state.merge({pgpPublicKey: action.payload.publicKey})
+    case ProfileGen.updatePromptForStore:
+      return state.merge({promptForStore: action.payload.promptForStore})
```

Comment:
Could this field have a more specific name? `promptForStore` is confusing because of the `store`.